src/commands/runtime/action/get.js

```diff
@@ -34,14 +34,17 @@ class ActionGet extends RuntimeBaseCommand {
         const opts = ow.actions.client.options
         const webFlag = result.annotations.find(elem => elem.key === 'web-export')
         const webAction = webFlag !== undefined && webFlag.value === true
+        const namespaceAsSubdomainSupported = opts.api.includes('https://adobeioruntime.net/api')
 
         let [namespace, packageName] = result.namespace.split('/')
         const actionPartOfPackage = !!packageName
 
         if (webAction) {
           const web = 'web'
           packageName = actionPartOfPackage ? packageName : 'default'
-          this.log(`${opts.api}${web}/${namespace}/${packageName}/${result.name}`)
+          namespaceAsSubdomainSupported
```

Comment:
It would be nice if this was factored into a class function so downstream projects could override this more conveniently.

Reply:
Thanks for the review @rabbah, could you expand a bit more on what you have in mind for this. I was thinking that ultimately this could be passed in as a config via .aio (or wskprops)

Reply:
@rabbah great suggestion. We'd release this PR on Wednesday, would you then be keen on taking over and making it configurable?

Reply:
Will do. 👍

Reply:
Great, thanks @rabbah !